### PR TITLE
Install statuspage

### PR DESCRIPTION
**Pull request by MarquiseRosier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "nyc": "15.0.1",
     "proxyquire": "2.1.3",
     "semantic-release": "17.0.6",
+    "statuspage": "^0.1.6",
```
Comment (kptdobe) [inline package.json:54]:
```suggestion
    "statuspage": "0.1.6",
```

other dependencies are strict...

**Comment by rofe** [comment]
@MarquiseRosier what's this for? I don't think we use this anywhere. Our Statuspage integration is in `helix-ops`...

**Comment by MarquiseRosier** [comment]
> @MarquiseRosier what's this for? I don't think we use this anywhere. Our Statuspage integration is in `helix-ops`...

hmmm i slightly disagree, circleci is asking for it in the project node_modules/.bin/statuspage, am I wrong? 

it's asking for it to be in the following directory: 

```
Error: Cannot find module '/home/circleci/project/node_modules/.bin/statuspage'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
```

And /home/circleci/project/ is basically 
```
if [ -e /home/circleci/project/.git ]
then
  cd /home/circleci/project
  git remote set-url origin "$CIRCLE_REPOSITORY_URL" || true
else
  mkdir -p /home/circleci/project
  cd /home/circleci/project
  git clone "$CIRCLE_REPOSITORY_URL" .
fi
```

And $CIRCLE_REPOSITORY_URL is git@github.com:adobe/helix-pages.git

So that's why I think helix-pages needs to have statuspage installed.
Unless there is more to this than I'm thinking! :)

**Comment by rofe** [comment]
> Unless there is more to this than I'm thinking! :)

Actually, less :) 

> `home/circleci/projectnode_modules/.bin/statuspage`

This is still our own tool, not something Statuspage provides. Note the `.bin` in there - this means it is a shortcut to a binary exposed by one of the node moduels. In this case it's provided by `helix-ops`: the source code is [here](https://github.com/adobe/helix-ops/blob/master/src/statuspage.js) and it is exposes under `.bin` [here](https://github.com/adobe/helix-ops/blob/master/package.json#L7).

Probably this error just means that `npm ci` isn't run anywhere in the `publish` job prior to the monitoring command, which would explain why the binary is missing.

Either way, it has nothing to do with the npm module `statuspage`.

**Comment by rofe** [comment]
Yep, pretty sure adding `setup` as a required step to `publish` [here](https://github.com/adobe/helix-pages/blob/2735b3557a5b693232242bb5933fadeeea9aabce/.circleci/config.yml#L295) will make this error go away:

File: package.json (hunk 1 above)
Comment (rofe) [inline package.json:54]:
```suggestion
```

**Comment by rofe** [comment]
My bad! The required *job* is actually `build`, which runs the `setup` *command* 😬 

Hint: you can run `circleci config validate .circleci/config.yml` to do a sanity check on the CircleCI config  before committing.

**Comment by rofe** [comment]
Ok, now monitoring automation should work when the `publish` job will run on master.

**Pull request merged**